velox/exec/Task.cpp

```diff
@@ -1419,10 +1419,13 @@ TaskStats Task::taskStats() const {
   // (their operators).
   TaskStats taskStats = taskStats_;
 
+  taskStats.totalDrivers = drivers_.size();
+
   // Add stats of the drivers (their operators) that are still running.
   for (const auto& driver : drivers_) {
     // Driver can be null.
     if (driver == nullptr) {
+      ++taskStats.completedDrivers;
```

Comment:
Not sure if it is better to update TaskStats the same way as we do for split except the blocked driver stats update @mbasmanova. Either way is fine to me.

Reply:
I was thinking that it is good to have all driver counters in one place, rather than scattered around the code.

Reply:
This might be an example: https://www.internalfb.com/code/fbsource/[c6cb459e3498d2db63e8788192cea1538213d63a]/fbcode/velox/exec/tests/TaskTest.cpp?lines=630. We can chat offline. thanks!